assets/js/googlesitekit/datastore/user/tracking.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * `core/user` data store: Tracking info.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from './constants';
+import { createFetchStore } from '../../data/create-fetch-store';
+import { actions as errorStoreActions } from '../../data/create-error-store';
+const { receiveError, clearError } = errorStoreActions;
+
+const fetchStoreReducerCallback = ( state, tracking ) => ( { ...state, tracking } );
+
+const fetchGetTrackingStore = createFetchStore( {
+	baseName: 'getTracking',
+	controlCallback: () => {
+		return API.get( 'core', 'user', 'tracking' );
+	},
+	reducerCallback: fetchStoreReducerCallback,
+} );
+
+const fetchSaveTrackingStore = createFetchStore( {
+	baseName: 'saveTracking',
+	controlCallback: ( enabled ) => API.set( 'core', 'user', 'tracking', { enabled: !! enabled } ),
+	reducerCallback: fetchStoreReducerCallback,
+	argsToParams: ( enabled ) => enabled,
+} );
+
+const baseInitialState = {
+	tracking: undefined,
+	isSavingTracking: false,
+};
+
+// Actions
+const SET_USER_TRACKING_SAVING_FLAG = 'SET_USER_TRACKING_SAVING_FLAG';
+
+const baseActions = {
+	/**
+	 * Saves user tracking settings.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {boolean} enabled Tracking status.
+	 * @return {Object} Object with `response` and `error`.
+	 */
+	*saveUserTracking( enabled ) {
```

Comment:
This name conflicts with the ACs where this is outlined as `setTrackingEnabled`. Let's go with that as described there. Alternatively, we could define our actions (and selectors) more closely to the REST endpoint, e.g.:
* `setTracking( data )` as low-level (private) action
* utility action `enableTracking()` which calls `setTracking( { enabled: true } )`
* utility action `disableTracking()` which calls `setTracking( { enabled: false } )`
* same for the selectors, we could have `getTracking()` which returns the full object, and `isTrackingEnabled()` which only returns the `enabled` property value

Anyway, let's use "set" throughout here instead of "save", as there is no need for differentiation, and "set" is a more intuitive action name that we use in most places.

Reply:
Updated to use names from the ACs.